Source package : scapy
File name       : scapy-2.5.0.tar.gz
Lookaside URL   : https://src.fedoraproject.org/repo/pkgs/scapy/scapy-2.5.0.tar.gz/sha512/4e5cacff0bbf6fd991ea24f4680049d042082fae4b349c8082078e6f01b38c09bb5c8276bfba15d4a88a1eb5af92c505848ec98556b10eecbd803f134a5b244a/scapy-2.5.0.tar.gz
Source URL      : https://github.com/secdev/scapy/archive/v2.5.0.tar.gz#/scapy-2.5.0.tar.gz

source file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/source-scapy-2.5.0.tar.gz: gzip compressed data, from Unix, original size modulo 2^32 19230720
lookaside file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/lookaside-scapy-2.5.0.tar.gz: gzip compressed data, from Unix, original size modulo 2^32 19230720


diff -r source-scapy-2.5.0.tar.gz-extracted/scapy-2.5.0/scapy/__init__.py lookaside-scapy-2.5.0.tar.gz-extracted/scapy-2.5.0/scapy/__init__.py
98c98
<     git_archive_id = '9473f77d8b5 v2.5.0'.strip().split()
---
>     git_archive_id = '9473f77d8b v2.5.0'.strip().split()
